src/language-css/parser-postcss.js

```diff
@@ -375,6 +375,18 @@ function parseNestedCSS(node) {
         return node;
       }
 
+      if (lowercasedName === "import") {
+        if (params.includes("#{")) {
```

Comment:
Don't use this, `postcss-values-parser` works great with `#{$var}`

Reply:
<img width="454" alt="screen shot 2018-05-12 at 10 30 21 pm" src="https://user-images.githubusercontent.com/11256141/39959751-234406aa-5634-11e8-98c5-29db20ef61d3.png">

It was adding a comma why I added this check here @evilebottnawi 
should I remove this check from here and try fixing it in `printer-postcss.js file` ?

Reply:
@ad1992 it is very strange, here only `@import` check, code around `media` below and we have check on `#{`

Reply:
@evilebottnawi my bad. this is fixed. 
Though all the test cases except for comments pass now but there are still some issues on how the postcss value parser behaves when having `,` in import statement .  i am working on fixing it.
For example testcase like
`@import url("very-very-very-very-very-very-very-very-very-very-long-name.css") projection,tv,tv1;
`
output of media query parser
```css
@import url("very-very-very-very-very-very-very-very-very-very-long-name.css")
  projection,
  tv,
  tv1;
```
Output of post css value parser
```css
@import url("very-very-very-very-very-very-very-very-very-very-long-name.css")
  projection,
  tv,  tv1;
```